src/eterna/ui/help/HelpToolTip.ts

```diff
@@ -0,0 +1,119 @@
+import {ContainerObject} from 'flashbang';
+import {Graphics, Point, Rectangle} from 'pixi.js';
+import Fonts from 'eterna/util/Fonts';
+
+export type ToolTipPositioner = [() => Rectangle, number];
+
+type HelpToolTipSide = 'top' | 'bottom';
+
+interface HelpToolTipProps {
+    text: string;
+    positioner: ToolTipPositioner;
+    side?: HelpToolTipSide;
+    tailLength?: number;
+    content?: PIXI.Container;
+}
+
+export default class HelpToolTip extends ContainerObject {
```

Comment:
Should this be using TextBalloon? https://github.com/eternagame/EternaJS/blob/2795295c89d3ebfe8ef5cf2af91e671f6d7df100/src/eterna/ui/TextBalloon.ts

Reply:
The help tooltips have a lot of specific functionality to warrant their own implementation (they can be attached to both the bottom or top of an element, they can have a tail of X length or not, they can host either a text of a container like the palette of the nucleotides). `TextBalloon` seems to have a lot of history (For ex. it has a 'Next' button) and would need heavy modifications to handle these. Also not sure how to make TextBallon look exactly like the tooltip in the comp.